### PR TITLE
Webpack improvements 

### PR DESCRIPTION
**Pull request by lneves12**
## What does this PR change?

This PR disable the regeneration of javascript licenses on watch mode and add a new webpack alias, so we can easily import the shared react components inside the folder "src/components".

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed

- [x] **DONE**

## Test coverage
- No tests

- [x] **DONE**

### CODE REVIEW

**Comment by lneves12** [comment]
gitarro no changelog needed !!!

File: web/html/src/build/webpack.config.js [hunk 1]
```diff
@@ -41,6 +41,11 @@ module.exports = (env, argv) => ([
             }
         ]
     },
+    resolve: {
+      alias: {
+        components: path.resolve(__dirname, '../components/'),
```
Comment (cbosdo) [inline web/html/src/build/webpack.config.js]: couldn't we add also `utils` to the list ?

Comment (lneves12) [reply web/html/src/build/webpack.config.js]: sure :) I will do it. 

Even better would be to have a folder "shared" and then have "shared/components" and "shared/utils", but that's too much work to do right now

Comment (cbosdo) [reply web/html/src/build/webpack.config.js]: :+1:

**Pull request merged**